src/sail_lean_backend/sail_plugin_lean.ml

```diff
@@ -229,6 +229,13 @@ let start_lean_output (out_name : string) default_sail_dir =
       ^ " " ^ Filename.quote lean_sail_dir
       )
   in
+  let _ =
```

Comment:
Could we factorize this with the other copies? It's starting to be a lot of copy-paste no?